apps/api/src/app/portfolio/calculator/twr/portfolio-calculator.ts

```diff
@@ -227,11 +228,11 @@ export class TWRPortfolioCalculator extends PortfolioCalculator {
 
     const dateOfFirstTransaction = new Date(first(orders).date);
 
-    const unitPriceAtStartDate =
-      marketSymbolMap[format(start, DATE_FORMAT)]?.[symbol];
+    const endString = format(end, DATE_FORMAT);
+    const startString = format(start, DATE_FORMAT);
```

Comment:
I think this would probably be better as `endDateString`/`startDateString`, but those were already used in the other dates-loop at the end of the method. What do you think of renaming those at the end to `range[Start|End]DateString`, and these here at the top to `[end|start]DateString`?